packages/block-editor/src/components/colors-gradients/panel-color-gradient-settings.js

```diff
@@ -215,7 +215,7 @@ const PanelColorGradientSettingsMultipleSelect = ( props ) => {
 
 	const customGradients = useSetting( 'color.gradients.user' );
 	const themeGradients = useSetting( 'color.gradients.theme' );
-	const defaultGradients = useSetting( 'color.gradients.core' );
+	const defaultGradients = useSetting( 'color.gradients.default' );
```

Comment:
I guess this one might be a breaking change? Should we provide an alias or maybe it's fine for this one?

Reply:
I think it is fine origins were never exposed in the core.